lib/modules/dosomething/dosomething_campaign/dosomething_campaign.theme.inc

```diff
@@ -445,20 +445,39 @@ function dosomething_campaign_preprocess_action_page(&$vars, &$wrapper) {
   }
 
   // Add social share bar.
+  $custom_social_copy_variables = [
+    'custom_facebook_copy' => $campaign->variables['custom_facebook_copy'],
+    'custom_twitter_copy' => $campaign->variables['custom_twitter_copy'],
+    'custom_tumblr_copy' => $campaign->variables['custom_tumblr_copy'],
+  ];
+
+  foreach ($custom_social_copy_variables as $key => $value) {
+    if ($value) {
+      $$key = $value;
+    }
+    elseif (isset($campaign->fact_problem['fact'])) {
```

Comment:
@sbsmith86 I was thinking here that if there is no custom social copy (filled out in Custom Settings), populate the social post with the campaign's fact problem (as it does now). I don't think that campaign leads will always write custom copy so the default will be to just post the problem fact that most campaigns have I believe.